src/main/java/com/redhat/devtools/lsp4ij/console/actions/ApplyLanguageServerSettingsAction.java

```diff
@@ -80,7 +80,7 @@ public void update(@NotNull AnActionEvent e) {
             }
         }
 
-        e.getPresentation().setEnabled(modified);
+        e.getPresentation().setEnabled(modified && languageServerView.isValid());

```

Comment:
Perhaps user want to save an invalid language server defintion?

Reply:
I went with the same validation that creating a new LS has, should this be removed and allow saving an invalid configuration?

Reply:
Alright, that does make sense too. I'll change the validation on that

Reply:
Thanks!